### PR TITLE
Seed

### PR DESCRIPTION
**Pull request by willchan8**

### CODE REVIEW

File: database/generatePhotos.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+const generateListingPhotos = () => {
+  const photosArray = [];
+  const usedPhotos = [];
+
+  // Generate 5-10 photos per listing.
+  const numberOfPhotos = getRandomInt(5, 11);
+
+  // Add an arbitrary # of photo objects to an array.
+  for (let i = 0; i < numberOfPhotos; i += 1) {
+    const randomIndex = getRandomInt(0, photoURLs.length);
```
Comment (lenacha) [inline database/generatePhotos.js]: randomIndex looks confusing for me, 
I think that you'd better change your variable name to randomPhoto

File: database/index.js [hunk 2]
```diff
@@ -1,12 +1,23 @@
 const mongoose = require('mongoose');
+const helper = require('./generateListings');
```
Comment (lenacha) [inline database/index.js]: I think that you'd better change `helper` to more specific name.

**Comment by lenacha** [comment]
I think that if you change some of the confusing variable names, it will be good enough to merge to master! Good job!

File: database/generatePhotos.js (hunk 1 above)
Comment (willchan8) [reply database/generatePhotos.js]: Changed to `randomIndex` to `randomPhotos`

File: database/index.js (hunk 2 above)
Comment (willchan8) [reply database/index.js]: Changed `helper` to `seedDB`

**Pull request merged**